### ISSUE TITLE
Contact Us: Console Error

### ISSUE THREAD

**Issue by romatitan** [titan-cebu/BNT-Legal#8]
![image](https://cloud.githubusercontent.com/assets/12888048/9377041/3a1298be-4747-11e5-9517-55a6ba6a5abd.png)

**Comment by romatitan** [comment]
![image](https://cloud.githubusercontent.com/assets/12888048/9513519/994b6f96-4cc3-11e5-9070-73f669b067ec.png)

**Comment by christiancebudigital** [comment]
Please clear your cache.. no error found on our end...

**Comment by romatitan** [comment]
I've cleared the cache and check again the console.

Chrome - console error: Falied to load resource: net::ERR_BLOCKED_BY_CLIENT
The error still occurs at my end, but I've tried to check with other PC and chrome-browser and there's no error occurs.
Result: 1:3
For chrome browser its won't fix.

But still a console error occurs in IE browsers.
IE - console error: SCRIPTS: Access is denied and 3 warnings
Result: 3:3
![image](https://cloud.githubusercontent.com/assets/12888048/9538714/e78360e0-4d79-11e5-9635-5ebb3fd873ba.png)
